Makefile

```
@@ -4,64 +4,66 @@ start:
 	@./scripts/start-infra.sh
 
 compile:
-	@make -f ${P4C_REPO}/backends/ebpf/runtime/kernel.mk BPFOBJ=demo.o P4FILE=demo.p4 ARGS="-DPSA_PORT_RECIRCULATE=2" P4ARGS="--Wdisable=unused" psa
+	@p4c-ebpf --arch psa --Wdisable=unused -o demo.c demo.p4
+	@clang -O2 -g -c -DBTF -emit-llvm -o demo.bc demo.c
+	@llc -march=bpf -mcpu=generic -filetype=obj -o demo.o demo.bc
 
 deploy:
-	@sudo nsenter --net=/var/run/netns/switch psabpf-ctl pipeline load id 1 demo.o
+	@nsenter --net=/var/run/netns/switch nikss-ctl pipeline load id 1 demo.o
 	@echo "Adding ports to switch..."
-	@sudo nsenter --net=/var/run/netns/switch psabpf-ctl add-port pipe 1 dev psa_recirc
-	@sudo nsenter --net=/var/run/netns/switch psabpf-ctl add-port pipe 1 dev eth0
-	@sudo nsenter --net=/var/run/netns/switch psabpf-ctl add-port pipe 1 dev eth1
-	@sudo nsenter --net=/var/run/netns/switch psabpf-ctl add-port pipe 1 dev eth2
+	@nsenter --net=/var/run/netns/switch nikss-ctl add-port pipe 1 dev psa_recirc
+	@nsenter --net=/var/run/netns/switch nikss-ctl add-port pipe 1 dev eth0
+	@nsenter --net=/var/run/netns/switch nikss-ctl add-port pipe 1 dev eth1
+	@nsenter --net=/var/run/netns/switch nikss-ctl add-port pipe 1 dev eth2
 	@echo "Ports successfully added"
-	@sudo scripts/routing.sh
+	@scripts/routing.sh
 	@echo "Done"
 
 clean:
-	@sudo nsenter --net=/var/run/netns/switch psabpf-ctl pipeline unload id 1 || true
+	@nsenter --net=/var/run/netns/switch nikss-ctl pipeline unload id 1 || true
 	@./scripts/stop-infra.sh || true
 	@rm demo.c demo.o demo.bc || true
```

What about to add `-f` option? Inside container behaves ok, but outside it will ask before removal
```suggestion
	@rm -f demo.c demo.o demo.bc || true
```